index.js

```diff
@@ -1,13 +1,99 @@
+const $closeMenuButton = document.querySelector('.nav-links img');
+const $linksMenu = document.querySelectorAll('.nav-links li a');
+const $modal = document.querySelector('.modal');
+const $modalTitle = document.querySelector('#modal-title');
+const $pModal = document.querySelector('#p-modal');
+const $imgModal = document.querySelector('#img-modal');
+const $liModal = document.querySelectorAll('.li-modal');
+const $openModal = document.querySelectorAll('.button-project');
+const $closeModal = document.querySelector('.close-button');
+const frstImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic-mask.png';
+const scndImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post-mask.png';
+const thrdImagePath = '/resources/work-section/masks/Snapshoot PortfolioTonic2-mask.png';
+const frthImagePath = '/resources/work-section/masks/Snapshoot PortfolioMulti-Post2-mask.png';
+let scrollEnabled = true;
+const projectsInfo = [{
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frstImagePath,
+}, {
+  name: 'Multi Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: scndImagePath,
+}, {
+  name: 'Tonic',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: thrdImagePath,
+}, {
+  name: 'Multi-Post Stories',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  technologies: ['HTML', 'CSS', 'Javascript'],
+  image: frthImagePath,
+}];
 
-let div = document.createElement('div');
-div.id = 'content';
-div.className = 'note';
+function selectProject(buttonIndex) {
```

Comment:
- [ ] KIndly incorporate the blurry effect beneath your popup to match the design 💯.

- Your design
<img width="960" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/10336337-7840-42cd-a139-cf500ec6d8fd">

- Figma design
<img width="263" alt="image" src="https://github.com/SaveryIV/SaveryIV.github.io/assets/41163571/9dd7b089-253b-4aaf-8aec-97c0cad3808d">